### ISSUE TITLE
test model

### ISSUE THREAD

**Issue by WYY1312112** [zhang-can/ECO-pytorch#29]
Author, thank you very much for your work.
I have a question, I want to get the classification accuracy of the 101 class, but after the test my results have been like this, what do you think might be the problem?

[ 0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.
  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  1.
  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.
  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.
  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.
  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.  0.]
Accuracy 0.99%

Looking forward to your reply

**Comment by youngkyoonjang** [comment]
This is what I have experienced the same. And I found my mistake that I did not switch to evaluation mode. It works fine after putting the following:
  model.eval()
before the test loop.

**Comment by qingchuanhuajuan** [comment]
Hello, Can you share your test code? thank you very much!

**Comment by ycl1035621458** [comment]
Hello, Can you share your test code? thank you very much!

**Comment by xiaochaoLiu1017** [comment]
Have you solved the problem? I meet the same problem. Looking forward to your reply.

**Comment by JasonYangZi** [comment]
> Have you solved the problem? I meet the same problem. Looking forward to your reply.

Hello, Can you share your test code? thank you very much!
